### ISSUE TITLE
permission denied on fuse directory

### ISSUE THREAD

**Issue by mark-hahn** [atom/electron#3529]
After upgrading Atom from 1.1.0 to 1.2.0 I am unable to access fuse directories.  I'm getting permission errors.  It was suggested on the Atom forum that the problem might be an node upgrade in an electron upgrade.

Can I downgrade the node version loaded by electron as a test?  Can anyone suggest a work-around for the problem?  I'm stuck on Atom 1.1.0 until this is fixed because all our projects reside on an objectivefs fuse volume.

**Edit:** I just noticed that `apm -v` is showing node at  0.10.40. This is an old node version which is weird since you just upgraded node.  Is this the expected version?

**Comment by mark-hahn** [comment]
I did a test with the node version from atom 1.2.0 and node had no trouble writing to my fuse directory, so the problem must be in electron.

How to repeat this problem in just a few minutes...

- download http://sourceforge.net/projects/fuse/files/fuse-2.X/2.9.4/
- `cd fuse-2*`  you will be in this fuze install dir for the rest of these instructions
- `./configure --prefix=/usr`
- `make`
- `mkdir fuze-dir` to make a dir to mount the fuze file system on
- `sshfs localhost:/$home fuze-dir` to mount `$home` on `fuze-dir`
- `ls -al fuze-dir` to see mount worked and permissions correct
- `atom fuze-dir` with any atom after 1.1.0 and see permissions error
- `umount fuze-dir` when done 

BTW, I found sshfs when creating these instructions but it looks like a very useful tool in general.  It can mount any remote directory as a local one.

**Comment by DerNivel** [comment]
`apm -v` prints out the version of `apm` and not of node.
https://github.com/atom/apm/blob/master/src/apm-cli.coffee#L75

**Comment by mark-hahn** [comment]
Must I assume this is low-priority and will not be fixed any time soon?  I need to know if I need to develop a new workflow.

**Comment by mark-hahn** [comment]
(Copy from atom/atom issue  https://github.com/atom/atom/issues/9555).

I have found out that the parents of fuse directories had 600 permissions instead of the usual 755.  This means that any other user wouldn't be able to list the directory I was trying to open with Atom.  But I am always logged in as root and I always start atom with root.  So I think this is the wrong behavior.  Atom shouldn't behave like a different user.  Also it didn't have this problem until after 1.1.0.

In any case I have fixed my problem by setting all dirs to 755.  So I have a workaround.  If anyone wants I'll file another issue about this anomalous behavior but I'm closing this one.
